hedera-node/src/main/java/com/hedera/services/state/submerkle/CurrencyAdjustments.java

```diff
@@ -73,13 +70,13 @@ public int getVersion() {
 
 	@Override
 	public void deserialize(SerializableDataInputStream in, int version) throws IOException {
-		accountIds = in.readSerializableList(MAX_NUM_ADJUSTMENTS, true, EntityId::new);
```

Comment:
⚠️ Records are in state, so we need new `RELEASE_0240_VERSION` and migration code here. 👍

Reply:
Added migration code for deserialize method